src/utilities/use-deep-compare-ref.tsx

```diff
@@ -4,6 +4,18 @@ import isEqual from 'lodash/isEqual';
 type DependencyList = ReadonlyArray<unknown>;
 type Comparator = (a: DependencyList, b: DependencyList) => boolean;
 
+/**
+ * useDeepCompare is valuable to keep a consistent dependency list across reference changes
```

Comment:
changes.
```suggestion
 * Allows for custom or deep comparison of a dependency list. Useful to keep a consistent dependency list across reference changes. 
```

Reply:
Not 100% sold on my suggested description either. I just think that simply outlining the value may not be the best description?